### ISSUE TITLE
Implement Eclipse JNoSQL for Jakarta Data TCK standalone testing

### ISSUE THREAD

**Issue by KyleAure** [OpenLiberty/open-liberty#30015]
Our Jakarta Data TCK test suite can run in standalone mode, but needs to use a standalone Jakarta Data provider. 

Implement the usage of the Eclipse JNoSQL provider as this standalone provider. 

See ignored test class:
https://github.com/OpenLiberty/open-liberty/blob/1fcf9eff5b7bb2e349f7050cf2c9a5e1811ed9df/dev/io.openliberty.jakarta.data.1.0_fat_tck/fat/src/io/openliberty/jakarta/data/tck/DataStandaloneTckLauncher.java#L38

**Comment by reshmismenon** [comment]
hi @KyleAure ,

Are you working on this issue? I was looking for an issue to get started.

**Comment by KyleAure** [comment]
@reshmismenon
No I have not gotten to this issue yet.
If you do work on this feel free to open a pull request and add me as a reviewer.

**Comment by reshmismenon** [comment]
Hi @KyleAure sure, I shall start with it add you as reviewer

**Comment by KyleAure** [comment]
@reshmismenon  I have assigned this issue to you.  Yes please add me as a reviewer for any PR you open.

**Comment by reshmismenon** [comment]
Hi @KyleAure , The TCK launcher is already implemented right? So we just have to remove **@Ignore** annotation right? Actual implementation of Jakarta Data for Eclipse JNOSQL is not part of this ticket right?

**Comment by KyleAure** [comment]
@reshmismenon
Yes, I previously did some work to create a TCK launcher for JNoSQL.  However, it currently does not work. So in addition to removing the `@Ignore` annotation we also need to update to the latest version of JNoSQL and likely import the JNoSQL annotation processor in order to get all the tests passing.

**Comment by reshmismenon** [comment]
ok @KyleAure  thanks for the information. I shall make those changes.

**Comment by reshmismenon** [comment]
Hi @KyleAure ,
I have pushed my changes and raised PR: https://github.com/OpenLiberty/open-liberty/pull/30596. I am trying to fix some errors in my local builds.

**Comment by reshmismenon** [comment]
Hi @KyleAure , I ran the following commands:

./gradlew cnf:intialize
./gradlew assemble
./gradlew build io.openliberty.jakarta.data.1.0_fat_tck:buildandrun

I am getting the failure in another suit file:///Users/reshmismenon/Documents/OpenLibertyCode/forkedRepo/open-liberty/dev/io.openliberty.netty.internal.impl/build/libs/reports/tests/test/index.html

Should we consider this?

<img width="1728" alt="Image" src="https://github.com/user-attachments/assets/5faf020c-5f9d-42b4-a5ea-e06d8219d47b" />

**Comment by KyleAure** [comment]
@reshmismenon Please refer to this wiki: https://github.com/OpenLiberty/open-liberty/wiki/Delivery-Process#step-1--development

If this is your first time building liberty you need to use: `./gradlew releaseNeeded`
Note that by doing `./gradlew cnf:intialize` you have wiped out the `dev/build.image/wlp` directory and therefore you NEED to run `./gradlew releaseNeeded` again.
After that point you can use `./gradlew assemble`
If you are still having issues here are some troubleshooting steps: 
1. You have an old cached gradle daemon with an incorrect change map. Try stopping all gradle daemons using `gradle --stop`
2. We also now run all builds in parallel, but some projects do not correctly declare inter-project dependencies. If this bucket is the one always failing you might be able to work around the issue using `./gradlew --no-parallel ...`

Let me know if that worked.

**Comment by reshmismenon** [comment]
Sure , thank you @KyleAure , i shall try that and update.

**Comment by reshmismenon** [comment]
Hi @KyleAure , I had executed the following steps:
1. ./gradlew cnf:initialize
2. ./gradlew releaseNeeded
3. ./gradlew assemble
4. /gradlew --no-parallel build io.openliberty.jakarta.data.1.0_fat_tck:buildandrun
Still I am getting this error:

<img width="1454" alt="Image" src="https://github.com/user-attachments/assets/155db0a9-78f6-4a48-b227-8b6a52107f34" />

<img width="1454" alt="Image" src="https://github.com/user-attachments/assets/5fa1a777-07f0-48cf-8488-fb12f4eec9c4" />

When I am running without the changes in integration branch as well I am getting the same error.
